assets/api/input_method.js

```diff
@@ -255,9 +257,6 @@
     this.dispatchEvent(evt);
   };
 
-  InputContext.prototype.getText = function() {
```

Comment:
I believe we still keep the interface, just that it will simply return `this.text` if no arguments are passed.

To implementing argument handling, check `forms.js` on DXR.